annotator-core/src/main/java/edu/ucr/cs/riple/core/metadata/field/FieldDeclarationStore.java

```diff
@@ -40,6 +42,12 @@ public class FieldDeclarationStore extends MetaData<FieldDeclarationInfo> {
    */
   public static final String FILE_NAME = "class_info.tsv";
 
+  /**
+   * A map from class flat name to a set of field names that are declared in that class but not
+   * initialized at declaration.
+   */
+  private Map<String, Set<String>> uninitializedFields;
```

Comment:
Do we use Guava in the annotator or not?  If we do use Guava, we could use a [MultiMap](https://guava.dev/releases/19.0/api/docs/com/google/common/collect/Multimap.html) here instead of managing the `Set` values ourselves

Reply:
Thank you for noticing this. Yes, we do and we are using `MultiMap` in other places. This should have been `MultiMap` as well. 217ec2faae678af3a0efb253c19402a7938c26d2